src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiRequestBodyGenerator.cs

```diff
@@ -66,15 +66,17 @@ public static OpenApiRequestBody CreateRequestBody(this ODataContext context, IE
                 Properties = new Dictionary<string, OpenApiSchema>()
             };
 
+            bool allParamsNullable = true;
             foreach (var parameter in action.Parameters.Skip(skip))
             {
+                allParamsNullable = parameter.Type.IsNullable;
```

Comment:
Won't this only pass the value of the last parameter that is evaluated in the loop? We probably need to do a bitwise comparison here

Reply:
Oh yes. You are right there